azure/durable_functions/models/TaskOrchestrationExecutor.py

```diff
@@ -165,14 +165,14 @@ def parse_history_event(directive_result):
 
             # We provide the ability to deserialize custom objects, because the output of this
             # will be passed directly to the orchestrator as the output of some activity
-            if (event_type == HistoryEventType.SUB_ORCHESTRATION_INSTANCE_COMPLETED 
-                and directive_result.Result is not None):
+            if (event_type == HistoryEventType.SUB_ORCHESTRATION_INSTANCE_COMPLETED and
+                directive_result.Result is not None):
                 return json.loads(directive_result.Result, object_hook=_deserialize_custom_object)
-            if (event_type == HistoryEventType.TASK_COMPLETED 
-                and directive_result.Result is not None):
+            if (event_type == HistoryEventType.TASK_COMPLETED and
+                directive_result.Result is not None):
                 return json.loads(directive_result.Result, object_hook=_deserialize_custom_object)
-            if (event_type == HistoryEventType.EVENT_RAISED
-                and directive_result.Result is not None):
+            if (event_type == HistoryEventType.EVENT_RAISED and
+                directive_result.Result is not None):
```

Comment:
This one needs to be `directive_result.Input` since that's what we're trying to deserialize in line 177.